problems/euler/multiplesOf3And5/java/jailarie/eulerProblemOne.java

```diff
@@ -0,0 +1,15 @@
+
+public class eulerProblemOne {
+	public static void main(String[] args) {
+		int n = 1000;
+		int current = 0;
+		int sum = 0;
+		
+		while( current < n) {
+			if(current % 3 == 0 || current % 5 == 0) {
+				sum += current;
+			}
+			current++;
+		}System.out.println("The sum of multiples of 3 and 5 below " + n + " is " + sum);
+	}	
+}
```

Comment:
Code looks good! Though the problem specifies to do this operation for numbers under 1000, I think the program could be improved/extended by adding functionality to let the user set the value of n. Would help with unit testing, too.

Reply:
I had thought about doing that originally, but was trying to stick to the original problem. Made the change for unit testing. Thanks @cjyovich !

Reply:
Nice!